src/test/java/com/crud/tasks/mapper/TaskMapperTestSuite.java

```diff
@@ -0,0 +1,66 @@
+package com.crud.tasks.mapper;
+
+import com.crud.tasks.domain.Task;
+import com.crud.tasks.domain.TaskDto;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
```

Comment:
is it spring really required to run this test? mapper can be instantiated by calling constructor instead of injecting it from spring context. This test should be much more faster if spring will be removed from it.